client/pages/ForgotPassword/index.js

```diff
@@ -0,0 +1,103 @@
+import React, { useContext, useState } from 'react';
+import { Formik } from 'formik';
+import axios from 'axios';
+import InputForm from '@components/InputForm';
+import Button from '@components/Button';
+import { ForgotPasswordValidator } from '@clientValidators/Auth';
+
+const ForgotPassword = () => {
+    const [errorState, setErrorState] = useState(false);
+    const [errorMessage, setErrorMessage] = useState('');
+    const [successState, setSuccessState] = useState(false);
+    const [successMessage, setSuccessMessage] = useState('');
+
+    const handleError = response => {
+        response
+            ? setErrorMessage(response)
+            : setErrorMessage('Network Error!');
+        setErrorState(true);
+    };
+
+    const handleSuccess = response => {
+        setSuccessMessage(response);
+        setSuccessState(true);
+    };
+
+    return (
+        <div
+            className="bg-no-repeat bg-center bg-cover  flex flex-col justify-center items-center sm:h-screen"
+            style={{ background: `url(/images/12.jpg)` }}
+        >
+            <div className="relative inline-block px-0 py-5 sm:w-9/12 max-w-custom w-500">
+                <h1 className=" text-5xl text-gray-550 text-center my-10 mt-0">
+                    Forgot Password
+                </h1>
+                <Formik
+                    initialValues={{
+                        email: ''
+                    }}
+                    validationSchema={ForgotPasswordValidator}
+                    onSubmit={(values, { setSubmitting, resetForm }) => {
+                        setErrorState(false);
+                        setSuccessState(false);
+                        axios
+                            .post('/api/v1/auth/reset', values)
+                            .then(res => {
+                                resetForm({
+                                    email: ''
+                                });
+                                handleSuccess(res.data.data.message);
+                                setSubmitting(false);
+                            })
+                            .catch(({ response }) => {
+                                handleError(response.data.message[0].message);
+                                setSubmitting(false);
+                            });
+                    }}
+                >
+                    {({
+                        values,
+                        errors,
+                        touched,
+                        handleChange,
+                        handleBlur,
+                        handleSubmit,
+                        isSubmitting
+                    }) => (
+                        <form
+                            onSubmit={handleSubmit}
+                            className="shadow-custom px-16 py-5 pb-8 sm:w-12/12 sm:rounded-54 bg-white text-center"
+                        >
+                            {successState && (
```

Comment:
Pull this out too since we aren't using success messages. We'll use a global toast